Userland/Libraries/LibMedia/Color/ColorConverter.cpp

```diff
@@ -76,6 +76,7 @@ DecoderErrorOr<ColorConverter> ColorConverter::create(u8 bit_depth, CodingIndepe
 
     // https://kdashg.github.io/misc/colors/from-coeffs.html
     switch (input_cicp.matrix_coefficients()) {
+    case MatrixCoefficients::BT470BG:
     case MatrixCoefficients::BT709:
```

Comment:
It looks like both [System B/G (PAL)](https://en.wikipedia.org/wiki/PAL#Colorimetry) and [System M (S170M, NTSC?) included in BT.1700](https://www.itu.int/rec/R-REC-BT.1700-0-200502-I/en) want to use the luminance values for BT.601, rather than BT.709.

(Feel free to move the BT.601 coefficients up if you want to, as it preceded BT.709 by quite a few years.)

Reply:
Ok that makes sense. I've moved it up.